iBudget/ibudget/views.py

```diff
@@ -20,10 +20,11 @@ def post(self, request):# pylint: disable=no-self-use
         try:
             pic = request.FILES['icon']
             if AwsService.check_if_in_bucket(pic):
-                AwsService.upload(pic)
-                pic = str(pic)
-                url = AwsService.get_image_url(pic)
-                return HttpResponse(url)
+                if AwsService.upload(pic):
+                    pic = str(pic)
```

Comment:
There is no need in this local variable and in general, such kind of variable is a bad smell in a code.